src/main/java/org/spongepowered/common/mixin/core/data/MixinCustomDataHolder.java

```diff
@@ -107,9 +108,9 @@ public boolean hasManipulators() {
     @Override
     public List<DataManipulator<?, ?>> getCustomManipulators() {
         final List<DataManipulator<?, ?>> list = Lists.newArrayList();
-        for (DataManipulator<?, ?> manipulator : this.manipulators) {
-            list.add(manipulator.copy());
-        }
+        list.addAll(this.manipulators.stream()
+                .map(DataManipulator::copy)
```

Comment:
Lambdas are mostly syntactic sugar for anonymous inner classes. Mixins don't support these as it would be too complicated. The inner class gets discarded with it and you get a class not found exception at runtime.